daemon/graphdriver/aufs/aufs_test.go

```diff
@@ -574,9 +560,8 @@ func TestStatus(t *testing.T) {
 	}
 
 	status := d.Status()
-	if status == nil || len(status) == 0 {
-		t.Fatal("Status should not be nil or empty")
-	}
+	assert.NotNil(t, status)
```

Comment:
Does `NotNil` also take `len(status) == 0` properly into account? i.e., in this example, only the second "if" is true; https://play.golang.org/p/CF37XvmSn9

```go
foo := [][2]string{}

if foo == nil {
	fmt.Println("I am nil")
}
if len(foo) == 0 {
	fmt.Println("I have zero items")
}
```

Reply:
Ah, right a `nil` slice is both, but an empty slice is not nil.

I'll change this to `assert.Len()`